src/tests/test_sysargv.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import sys
+
+
+class SysArgvTests(unittest.TestCase):
+    """Test sys.argv state."""
+    def testSysArgvState(self):
+        """Test sys.argv state."""
+        argv = list(sys.argv) # need to make a copy of list
+        import clr
+        self.assertTrue(argv == sys.argv)
```

Comment:
I have no idea what I was trying to write... I meant to say, that since other tests are doing `import clr` before this test, the library is already loaded into memory and won't be reimported and the `import clr` statement may do nothing.